styles/modules.css

```diff
@@ -53,13 +65,13 @@ form input{
 
 #create-new-post{
   position: fixed;
-  right:-330px;
+  right:-28%;
```

Comment:
This negative margin makes the new post form a little difficult to use. (Really like the concept though, and I look forward to seeing the form with full functionality.) This could be interesting instead of using a negative margin:

```
position: fixed;
right: 0;
margin-right: 100px;
````
I might be totally missing the functionality though.